.github/workflows/velox_be.yml

```diff
@@ -180,7 +174,7 @@ jobs:
       - name: Build and Run unit test for Spark 3.3.1(slow tests)
         run: |
           docker exec velox-ubuntu2004-test-spark33-slow-$GITHUB_RUN_ID bash -l -c 'cd /opt/gluten && \
-          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut -DargLine="-Dspark.test.home=/var/cache/spark33/spark331" -DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest'
+          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut -DargLine="-Dspark.test.home=/opt/spark322" -DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest'
```

Comment:
I'm not sure on this, as there are some difference on spark33 vs spark32 on unit tests

Reply:
It looks the new test failure comes from the inconsistent spark test home. So we cannot do this.
As mentioned in the meeting, it is better to directly use the pre-built spark kept in docker image, which can not only fix this test issue, but also accelerate the CI test.